README.md

```diff
@@ -62,6 +62,7 @@ if you're on campus use the `ubcsecure` or `resnet` networks for best results.
 - Only commit files that are essential for the system to run; do not put any photos or videos in here
 - All files **must** be formatted properly. Formatting will be enforced with the `clang-format` tool. 
     - To check and fix formatting, from the `Snowflake` folder run `./clang_format/fix_formatting.sh BRANCH_NAME`, where `BRANCH_NAME` is the name of the branch you intend to merge your code into (ex. `iarrc` or `core`). This script will fix any improperly formatted code, but will refuse to change any files with uncommited changes (to prevent you losing work)
+- Once your pull request has been reviewed and revised until it looks good from both your and the reviewers' sides, go ahead and Squash and Merge it, which will squash all the commits on your pull request into one and merge it to `core`.
```

Comment:
Quick note here, we could be merging to other branches, such as `iarrc` or `igvc`, so that last section: "merge it to `core`." might better be something like "and merge it into the target branch"